package.xml

```diff
@@ -15,6 +15,7 @@
   <exec_depend>ros2nodl</exec_depend>
   <exec_depend>ros2pkg</exec_depend>
```

Comment:
Does this package actually require `ros2pkg` in any way?

Reply:
Actually, the package doesn't require `ros2pkg`. This should also probably be part of some draft branch where `nodl_to_policy` can be supplied a package name instead of just a NoDL XML file. I would've used the `ros2pkg.api.package_name_completer` to help implement that feature.

Regardless - I've removed this dependency in 74c9fcc.